include/libsemigroups/fpsemi-examples.hpp

```diff
@@ -22,710 +22,1318 @@
-    //! author).
-    [[nodiscard]] inline author operator+(author auth1, author auth2) {
-      return static_cast<author>(static_cast<uint64_t>(auth1)
-                                 + static_cast<uint64_t>(auth2));
-    }
-
-    //! A presentation for the stellar monoid.
-    //!
-
-    //! the stellar monoid with `l` generators, as in Theorem 4.39 of
-    //! [10.48550/arXiv.1910.11740][].
-    //!
-    //! \param l the number of generators
-    //!
-    //! \returns A `std::vector<relation_type>`
-    //!
-    //! \throws LibsemigroupsException if `l < 2`
-    //!
-    //! [10.48550/arXiv.1910.11740]: https://doi.org/10.48550/arXiv.1910.11740
-    [[nodiscard]] Presentation<word_type> stellar_monoid(size_t l,
-                                                         author val
-                                                         = author::Any);
-
-    //! A presentation for the dual symmetric inverse monoid.
-    //!
-    //! Returns a monoid presentation defining
-    //! the dual symmetric inverse monoid of degree `n`. The argument `val`
-    //! determines the specific presentation which is returned. The options are:
-    //! * `author::Easdown + author::East + author::FitzGerald` (from Section 3
-    //! of [10.48550/arxiv.0707.2439][])
-    //!
-    //! The default for `val` is the only option above.
-    //!
-    //! \param n the degree
-    //! \param val the author of the presentation
-    //!
-    //! \returns A `std::vector<relation_type>`
-    //!
-    //! \throws LibsemigroupsException if `n < 3`
-    //! \throws LibsemigroupsException if `val` is not
-    //! `author::Easdown + author::East + author::FitzGerald`
-    //!
-    //! [10.48550/arxiv.0707.2439]: https://doi.org/10.48550/arxiv.0707.2439
-    [[nodiscard]] Presentation<word_type> dual_symmetric_inverse_monoid(
-        size_t n,
-        author val = author::Easdown + author::East + author::FitzGerald);
-
-    //! A presentation for the uniform block bijection monoid.
-    //!
-    //! Returns a monoid presentation defining
-    //! the uniform block bijection monoid of degree `n`. The argument `val`
-    //! determines the specific presentation which is returned. The only option
-    //! is:
-    //! * `author::FitzGerald` (see [10.1017/s0004972700037692][])
-    //!
-    //! The default for `val` is the only option above.
-    //!
-    //! \param n the degree
-    //! \param val the author of the presentation
-    //!
-    //! \returns A `std::vector<relation_type>`
-    //!
-    //! \throws LibsemigroupsException if `n < 3`
-    //! \throws LibsemigroupsException if `val` is not `author::FitzGerald`
-    //!
-    //! [10.1017/s0004972700037692]: https://doi.org/10.1017/s0004972700037692
-    [[nodiscard]] Presentation<word_type>
-    uniform_block_bijection_monoid(size_t n, author val = author::FitzGerald);
-
-    //! A presentation for the partition monoid.
+    //! \name Specific presentations
```

Comment:
This (along with `\name Default presentations` further down in the file) constructs a *member group* which splits the documentation for the function definitions into two parts; one for the default functions (e.g. `partition_monoid`), and one for the specific presentations (e.g. `partition_monoid_HR05`). I think this is nice, but for some reason it means the documentation appears in the order it is in the `.hpp` file, rather than alphabetically. Is this okay? Alternatives are reverting to no member groups, sorting the source file, or doing something completely different such as defining new groups.

Reply:
Sort them to be alphabetical